src/containers/QAReviewContainer.jsx

```diff
@@ -122,11 +122,8 @@ class QAReviewContainer extends React.PureComponent {
     return rw;
   }
 
-  getOrderedPages(pages, id, investigation) {
-    return filter(pages, ['investigation', id || investigation]).sort(
-      (a, b) => a.order - b.order
```

Comment:
this method was performing a redundant sort on the pages, they are already sorted by the `order` property from graphQL and so after `sectionId` was added as the primary sort, this sort would undo that and revert it to sorting by `order`